### PR TITLE
fix: [CDS-78947]: GitOps SMP self signed certs

### PR DESCRIPTION
**Pull request by manavjot-harness**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: Harness GitOps Agent with self-signed certificates.
+description: This topic describes how to installGitOps Agent with self-signed certificates.
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md:3]: This topci describes how to install and configure Gitops Agent to connect to server with self signed certificate

File: docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+sidebar_position: 3
+redirect_from:
+  - /docs/continuous-delivery/gitops/harness-git-ops-agent-with-self-signed-certificates
+---
+
+Harness Self-Managed Enterprise Edition supports authorization by self-signed certificate. This topic describes how to use self-signed certificates with the Harness GitOps agent.
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md:9]: This is not an authorization,
lets put the same as above `This topic describes how to install and configure Gitops Agent to connect to server running with self-signed certificate`

File: docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+     ca.bundle: |  
+       -----BEGIN CERTIFICATE-----  
+       XXXXXXXXXXXXXXXXXXXXXXXXXXX  
+       -----END CERTIFICATE-------  
+       -----BEGIN CERTIFICATE-----  
+       XXXXXXXXXXXXXXXXXXXXXXXXXXX  
+       -----END CERTIFICATE-------
+   ```
+
+2. Add your certificates to the `ca.bundle` field.
```
Comment (mteodor) [inline docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md:37]: we should provide an example how to get `ca.bundle`, exporting from browser or using openssl command

**Review by mteodor — APPROVED** [review]

File: docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md (hunk 1 above)
Comment (manavjot-harness) [reply docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md]: done.

File: docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md (hunk 3 above)
Comment (manavjot-harness) [reply docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates.md:37]: done.

**Review by douglas-j-bothwell — APPROVED** [review]
Approving/merging without a review. Received a P0 request to merge this in immediately.

**Comment by douglas-j-bothwell** [comment]
**[8:44 AM](https://harness.slack.com/archives/C05RGD0MKDK/p1694187850280089)
Shiv**
 
[@Michael Cretzman (Docs)](https://harness.slack.com/team/UBHK2U9V0) heads up. we need to create a new page for gitops agent installation with self-signed certificate

**[@Manavjot Singh](https://harness.slack.com/team/U041GR8P926)**
 should have the content soon. this is urgent and needed by CBP customer. Need help on this. Thanks!

**[9:30 AM](https://harness.slack.com/archives/C05RGD0MKDK/p1694190635956409)
Manavjot Singh**
 please review: https://github.com/harness/developer-hub/pull/3254

**[11:53 AM](https://harness.slack.com/archives/C05RGD0MKDK/p1694199211842359)
Shiv**
 
[@Michael Cretzman (Docs)](https://harness.slack.com/team/UBHK2U9V0)
 can you merge the above PR asap? we need to send this to customer now. Thanks! (edited) 

**[11:55 AM](https://harness.slack.com/archives/C05RGD0MKDK/p1694199358511999)
douglas.bothwell**

 was added to the conversation by Shiv
.
**[11:56 AM](https://harness.slack.com/archives/C05RGD0MKDK/p1694199385485549)
Shiv**
 
[@douglas.bothwell](https://harness.slack.com/team/U030LJP00QN)
 can you merge the above PR? its reviewed and approved by our team. this is urgent. Thanks!

**[12:00 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694199638984399)
douglas.bothwell**
 
[@Shiv](https://harness.slack.com/team/U01124VC43C)
 I'm sorry our docs team is in the process of an extensive reorganization of the HDH and I've been asked to hold off merging in anything new today. I'm meeting with Ravi in 30m or so and should have a better estimate of when we can start merging in new updates.

**[12:01 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694199677930579)
Shiv**
 
[@douglas.bothwell](https://harness.slack.com/team/U030LJP00QN)
 this is P0 for CBP customer. we have to send this doc by noon PT

**[12:12 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200371512119)
douglas.bothwell**
 
[@Shiv](https://harness.slack.com/team/U01124VC43C)
 
While I can't merge in this PR right now, you should be able to view the updated content with these links:

- https://64fb5bd85e5dc331bb7bc062--harness-developer.netlify.app/docs/continuous-delivery/gitops/use-gitops/harness-git-ops-agent-with-self-signed-certificates
- https://64fb5bd85e5dc331bb7bc062--harness-developer.netlify.app/docs/continuous-delivery/gitops/use-gitops/install-a-harness-git-ops-agent

I believe these links should work outside of Harness...LMK if the CBP can't view these links and I'll escalate the request to merge this PR right away.

[12:15 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200547881849)
Shiv
 
**[@douglas.bothwell](https://harness.slack.com/team/U030LJP00QN)**
 whom should I involve to get approval to merge this PR?

**[12:18 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200688146679)
douglas.bothwell**
 
[@Ravi](https://harness.slack.com/team/UJQG0C0DR)

**[12:18 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200712933359)**
Ravi
 was added to the conversation by douglas.bothwell
.
**[12:19 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200773501939)**
Shiv
 I am driving

[@Manavjot Singh](https://harness.slack.com/team/U041GR8P926)
 can you and Ravi and request approval

**[12:21 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200862085079)**
Kim Fields
 was added to the conversation by douglas.bothwell
.
**[12:21 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200912339859)**
Ravi
 
[@douglas.bothwell](https://harness.slack.com/team/U030LJP00QN)
 feel free to merge it in since this is a P0. Sid's changes is a moving target.
New
**[12:22 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200948628429)**
Shiv
 Thanks 
[@Ravi](https://harness.slack.com/team/UJQG0C0DR)
:white_check_mark:
1

**[12:22 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694200952323649)**
Ravi
 I don't believe it should have impact.

**[12:23 PM](https://harness.slack.com/archives/C05RGD0MKDK/p1694201039602659)**
douglas.bothwell
 I just merged. I believe it takes 10-20m for updates to appear in the public HDH.

**Comment by douglas-j-bothwell** [comment]
[ DOC-3700 CD doc :: peer review of content in PR-3254](https://harness.atlassian.net/browse/DOC-3700)

**Pull request merged**